features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/trace/AndroidTracerTest.kt

```diff
@@ -650,7 +651,7 @@ internal class AndroidTracerTest {
             verify(mockSpan).log(capture())
             assertThat(firstValue)
                 .containsEntry(Fields.MESSAGE, anErrorMessage)
-                .containsOnlyKeys(Fields.MESSAGE)
```

Comment:
we need to put this assertion back as well. Basically we want to assert that there are only 2 keys.

Same for the similar tests below.